client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-layout.tsx

```diff
@@ -68,17 +65,79 @@ const PatternLayout = ( {
 							/>
 						</li>
 					) : (
-						<li className="pattern-layout__list-item pattern-layout__list-item--header">
-							<Button className="pattern-layout__list-item-button" onClick={ onAddHeader }>
-								<span
-									className={ classnames( 'pattern-layout__icon', 'pattern-layout__icon-add' ) }
-								>
-									+
-								</span>
+						<li className="pattern-layout__list-item">
+							<Button className="pattern-layout__add-button" onClick={ onAddHeader }>
+								<span className="pattern-layout__add-button-icon">+</span>
 								{ translate( 'Add a header' ) }
 							</Button>
 						</li>
 					) }
+					{ sections.length > 0 && (
+						<AsyncLoad require="./animate-list" featureName="domMax" placeholder={ <div /> }>
+							{ ( m: any ) =>
+								sections.map( ( { category, key }: Pattern, index ) => {
+									return (
+										<m.li
+											key={ key }
+											layout="position"
+											exit={ { opacity: 0, x: -50, transition: { duration: 0.2 } } }
+											className="pattern-layout__list-item"
+										>
+											<Icon className="pattern-layout__icon" icon={ group } size={ 24 } />
+											<span className="pattern-layout__list-item-text" title={ category }>
+												{ category }
+											</span>
+											<PatternActionBar
+												patternType="section"
+												onReplace={ () => onReplaceSection( index ) }
+												onDelete={ () => onDeleteSection( index ) }
+												onMoveUp={ () => onMoveUpSection( index ) }
+												onMoveDown={ () => onMoveDownSection( index ) }
+												enableMoving={ true }
+												disableMoveUp={ index === 0 }
+												disableMoveDown={ sections?.length === index + 1 }
+											/>
+										</m.li>
+									);
+								} )
+							}
+						</AsyncLoad>
+					) }
+					<li className="pattern-layout__list-item">
+						<Button className="pattern-layout__add-button" onClick={ () => onAddSection() }>
+							<span className="pattern-layout__add-button-icon">+</span>
+							{ translate( 'Add sections' ) }
+						</Button>
+					</li>
+					{ selectedFooter ? (
+						<li className="pattern-layout__list-item">
+							<Icon className="pattern-layout__icon" icon={ footer } size={ 24 } />
+							<span className="pattern-layout__list-item-text" title={ selectedFooter.category }>
+								{ selectedFooter.category }
+							</span>
+							<PatternActionBar
+								patternType="footer"
+								onReplace={ onReplaceFooter }
+								onDelete={ onDeleteFooter }
+							/>
+						</li>
+					) : (
+						<li className="pattern-layout__list-item">
+							<Button className="pattern-layout__add-button" onClick={ onAddFooter }>
+								<span className="pattern-layout__add-button-icon">+</span>
+								{ translate( 'Add a footer' ) }
+							</Button>
+						</li>
+					) }
+				</ul>
+			</div>
+		);
+	}
+
+	return (
```

Comment:
Display the new layout including the list of selected section patterns and the `+` button with the tooltip

![image](https://user-images.githubusercontent.com/13596067/220595117-226c493f-3ed6-438f-90de-06144582ffe3.png)